interview-questions.md

```diff
@@ -7,41 +7,42 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props are a little like paramaters/arguments in regular JavaScript. Props stand for properties. Props are how we access pieces of React state in other components. If we assign a variable to a piece of state within the componenet call, we can then access that piece of state later on in another component by using the React keyword "props"
 
-  Researched answer:
+
+  Researched answer: Props are arguments passed into React components. Props are passed to components via html attributes and stands for properties.
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event is how a developer affects change on a web page. An event could be a key press or lift, a mouse click or release as well as many other things.
 
-  Researched answer:
+  Researched answer: DOM events allow JavaScript to register various event handlers on various elements in an HTML page.
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: In functional programming, data isn't be stored in objects, it is only manipulated by other functions. In OOP, we store data in Objects and instances of those objects.
 
-  Researched answer:
+  Researched answer: Functional programming is done by writing pure functions. It does not store data, it manipulates it. OOP, stores data as objects and instances of the objects.
```

Comment:
While these answers are totally correct they are not compelling. The answers don't showcase any experience you have with the material. As an interviewer I know by this answer that you can google answers not that you have worked in OOP or FP. 

Try sharing a personal story of how you used OOP and one of the Pillars of OOP like inheritance. This will showcase a knowledge gained through experience and understanding.